src/status_im2/contexts/onboarding/common/syncing/view.cljs

```diff
@@ -0,0 +1,51 @@
+(ns status-im2.contexts.onboarding.common.syncing.view
```

Comment:
there is a syncing context -> status-im2.contexts.syncing... should this be there instead?